src/fluree/db/api.clj

```diff
@@ -341,28 +340,28 @@
   Attempts to use a ledger in a deletion state will throw an exception."
   ([conn ledger] (delete-ledger-async conn ledger))
   ([conn ledger opts]
-   (try (let [{:keys [nonce expire timeout private-key] :or {timeout 60000}} opts
-              timestamp (System/currentTimeMillis)
-              nonce     (or nonce timestamp)
-              expire    (or expire (+ timestamp 30000))     ;; 5 min default
-              cmd-data  {:type   :delete-db
-                         :db     ledger
-                         :nonce  nonce
-                         :expire expire}]
-          (if private-key
-            (let [cmd          (-> cmd-data
-                                   (util/without-nils)
-                                   (json/stringify))
-                  sig          (crypto/sign-message cmd private-key)
-                  persisted-id (submit-command-async conn {:cmd cmd
-                                                           :sig sig})]
-              persisted-id)
-            (ops/unsigned-command-async conn cmd-data)))
-        (catch Exception e e))))
+   (try
+     (let [{:keys [nonce expire timeout private-key] :or {timeout 60000}} opts
+             timestamp (System/currentTimeMillis)
+             nonce     (or nonce timestamp)
```

Comment:
extremely minor, but if you set the timestamp as the first let binding, then you can use it as the default for the `nonce` value in the destructuring form's `:or` map, and you can set the default `expire` value too. 

that will save you two whole lines!

Reply:
Not new code (I just moved it under the `try` so it wouldn't go so far to the right). But I touched it so I'll own it! :)